packages/malabi/src/exporter/jaeger.ts

```diff
@@ -0,0 +1,15 @@
+import { JaegerExporter } from '@opentelemetry/exporter-jaeger';
+
+const JAEGER_HOST = process.env.MALABI_JAEGER_HOST || 'localhost';
+export const jaegerExporter = new JaegerExporter({
+    tags: [],
+    host: JAEGER_HOST,
```

Comment:
prefer to configure using the opentelemetry environment variables:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#jaeger-exporter